### DISCUSSION TITLE
fast-typo-fix/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#7] [Ideas]
# fast-typo-fix/

曾经有人戏称——参与开源软件最快的方式就是给开源仓库修 Typo，这其实没问题。 不过也有人看不起修 Typo 的人，认为只是为了刷工作量，代码中的 Typo 有的

https://nova.moe/fast-typo-fix/

**Comment by iola1999** [comment]
> 拍摄于 Kong 上海办公室楼下

图片方向信息不正确。谢谢治好了颈椎病 (x

**Reply by n0vad3v** [reply]
哈哈哈感谢指出，已经修复~

**Comment by zdyxry** [comment]
之前在修改内部文档时，如果只是描述性文档还好，如果是 API 或者产品定义字段相关的文档有 typo，无法直接修改，修改了就和代码后 API 不一致，也涉及到了历史兼容性问题。

**Reply by n0vad3v** [reply]
确实，如果 Typo 和产品代码混合在一起的话处理起来难度对比文档而言就不是一个数量级了

**Comment by xu8117** [comment]
「Typo 仙人」 ，哈哈哈哈哈

**Reply by n0vad3v** [reply]
![image](https://github.com/n0vad3v/blog-comments/assets/24852034/55c791ef-5d72-427a-8d1c-c2ec58ec3a9d)

**Comment by tao12345666333** [comment]
离这么近，当然是欢迎上办公室来喝瓶快乐水了  :)

**Reply by n0vad3v** [reply]
好呀好呀，下次路过时看看能不能绕过楼下的门禁溜进来参观一下 😃😃
